[Structural][ConstitutiveLaws] Use InitialState for strain in SmallStrainIsotropicDamage

marandra opened the pull request:
**Description**
- Add usage of InitialState for strain to SmallStrainIsotropicDamage3D.
- Improve STRAIN calculation in ElasticIsotroipc3D.

**Changelog**
- ElasticIsotropic3D: Add CalculateValue(INITIAL_STRAIN_VECTOR)
- ElasticIsotropic3D: Compute STRAIN via CalculateMaterialResponsePK2
- SmallStrainIsotropicDamage3D: Add AddInitialStrainContibution to CalculateMaterialResponse
- SmallStrainIsotropicDamage3D: Compute STRAIN and INITIAL_STRAIN_VECTOR via ElasticIsotropic3D
- SmallStrainIsotropicDamage3D: Add other stress measures to FinalizeMaterialResponse and CalculateMaterialResponse
- Fix doc strings
- Minor: rename of variables, enhancement of the comments, code format

marandra committed:
Use InitialState for strain to SmallStrainIsotropicDamage

philbucher commented on applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp:
```diff
@@ -213,28 +212,50 @@ Vector& ElasticIsotropic3D::CalculateValue(
+    } else if (rThisVariable == INITIAL_STRAIN_VECTOR) {
+        if (this->HasInitialState()) {
+           rValue = GetInitialState().GetInitialStrainVector();
+        }
```
Maybe assign an empty Vector here?
Otherwise the output might be messed up I think

loumalouomega replied:
I am confused too

marandra replied:
You mean to set rValue to empty before the "if", right?

philbucher replied:
no an `else`
~~~c++
        if (this->HasInitialState()) {
           rValue = GetInitialState().GetInitialStrainVector();
        } else {
          rValue = ZeroVector(0);
        }
~~~

I am not sure though bcs of the usage

what do you think?

marandra replied:
Makes sense. Done.

RiccardoRossi replied:
what about the noalias?

philbucher commented on applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp:
```diff
@@ -213,28 +212,50 @@ Vector& ElasticIsotropic3D::CalculateValue(
 
-        // We compute the stress
-        ElasticIsotropic3D::CalculateMaterialResponseCauchy(rParameterValues);
+        ElasticIsotropic3D::CalculateMaterialResponsePK2(rParameterValues);
```
why do you changes this?

loumalouomega replied:
Idem

marandra replied:
PK2 is the measure actually coded, so calling:  Cauchy -> PK2, which is OK inside ElasticIsotropic3D, but for derived CLs (e.g. Damage) it calls Elastic::CalculateMaterialResponseCauchy -> Damage::CalculateMaterialREsponsePK2. 

So trying to get the strain in the DamageCL makes it a recursive call:
Damage::CalculateMaterialPK2 -> Elastic::CalculateStrain -> Elastic::CalculateMaterialCauchy -> Damage::CalculateMaterialPK2...

This change (PK2 for Cauchy) looked the less invasive.

philbucher commented on applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp:
```diff
@@ -213,28 +212,50 @@ Vector& ElasticIsotropic3D::CalculateValue(
+
+        // Previous flags restored
+        r_flags.Set( ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR, flag_const_tensor );
+        r_flags.Set( ConstitutiveLaw::COMPUTE_STRESS, flag_stress );
```
can you please explain why this is changed?

loumalouomega replied:
Idem

AlejandroCornejo replied:
I think that now, with the inclusion fo the initial strain, since the addition of this initial strain is done inside the calculateMaterialResponse, in order to print properly the strain is required to call the CalcMatResp....

marandra replied:
The original code directly jumping to the calculation of the strain in the CL. The code flow should be to check the USE_ELEMENT_PROVIDED_STRAIN flag, compute the strain in the CL if false, and increment it with the initial strain (if provided). This is already coded in the CalulateMaterialResponsePK2 function, so we are getting the strain from there, similarly to the way the stress is computed. Also not checking the USE_ELEMENT_PROVIDED_STRAIN flag was unexpected in the SmallStrainIsotropicDamagage3D CL and was throwing segfaults.

marandra replied:
(btw, GREEN_LAGRANGE_STRAIN_VECTOR and ALMANSI_STRAIN_VECTOR never reach this point, as they are intercepted in the base solid element. A fix to that requires fixing also some tests, as @loumalouomega  noted in #8183)

loumalouomega commented on applications/ConstitutiveLawsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp:
```diff
@@ -54,11 +54,12 @@ SmallStrainIsotropicDamage3D::~SmallStrainIsotropicDamage3D()
 bool SmallStrainIsotropicDamage3D::Has(const Variable<double>& rThisVariable)
 {
     if(rThisVariable == STRAIN_ENERGY){
```
Replace with else if, or use swicth (with more than 4 cases it is mor eefficient)

marandra replied:
Done

loumalouomega reviewed (changes requested):
Some requests

marandra committed:
Use 'else if'

marandra committed:
Use 'else if'

marandra committed:
Use ZeroVector if no initial state

SergioJimenezReyes committed:
Reverting changes finite strain plast and fixing exp plasticity

Reverting the changes done for stability in the finite strain plas law and fixing the minimum energy condition in exp plas

rubenzorrilla committed:
Add Eigen SVDs

Commit:
Merge pull request #8540 from KratosMultiphysics/core/tessellation_modeler_geom

rubenzorrilla committed:
Merge branch 'master' into fsi/two-fields-coupling

RiccardoRossi committed:
cleanup in the use of span

VeronikaSinger committed:
Update applications/ParticleMechanicsApplication/tests/test_search_mpm_particle_condition.py

Co-authored-by: tteschemacher <tteschemacher@users.noreply.github.com>

jgonzalezusua committed:
Renaming variables to make code more readable

SergioJimenezReyes committed:
Merge pull request #8453 from KratosMultiphysics/improving-stability-plasticity

[CconstitutiveLawsApp] Increase stability when Dissipation->1.0

joaquinirazabal committed:
[DEM] New processes to apply external forces and moments (#8533)

* Add new process for applying forces and moments

* Divide the process to apply forces and moments for spheres and walls

* Change process name to be shorter

* Revert tearDown classmethod

* Address philbucher suggestions

* Import parallel_utilities and pass parameters by value

* Add apply_forces_and_moments_process to loads_process_list

* Codacy

* Move new processes to KRATOS_DEM_APPLICATION_CORE

* Resolve Windows compilation

miguelmaso committed:
calling only if is output step

miguelmaso committed:
specifying printing times

miguelmaso committed:
skip test if scipy not available

miguelmaso committed:
more robust

VeronikaSinger committed:
Merge pull request #8498 from KratosMultiphysics/particle/fix_search_condition_update

[ParticleMechanicsApplication]/fix search condition geometry update

jgonzalezusua committed:
Merge pull request #8497 from KratosMultiphysics/Fix-error-in-phantoms

[DEM-SDEM] Fix error in phantoms

rubenzorrilla committed:
Merge branch 'master' into linearsolvers/eigen-svd-decomposition

sunethwarna committed:
use explicit

RiccardoRossi commented on applications/ConstitutiveLawsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp:
```diff
@@ -54,12 +54,13 @@ SmallStrainIsotropicDamage3D::~SmallStrainIsotropicDamage3D()
 {
     if(rThisVariable == STRAIN_ENERGY){
-        // explicitly returning "false", so we know we must call CalculateValue(...)
+        // explicitly returning "false", so the element calls CalculateValue(...)
```
as i understand it would return false anyway if you don't find it

RiccardoRossi commented on applications/ConstitutiveLawsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp:
```diff
@@ -72,6 +73,10 @@ bool SmallStrainIsotropicDamage3D::Has(const Variable<Vector>& rThisVariable)
+        
+    } else if(rThisVariable == STRAIN){
+        // explicitly returning "false", so the element calls CalculateValue(...)
+        return false;
```
i would say that this one is not needed. if it is not find, it will return false

RiccardoRossi commented on applications/ConstitutiveLawsApplication/custom_constitutive/small_strain_isotropic_damage_3d.cpp:
```diff
@@ -124,59 +129,85 @@ void SmallStrainIsotropicDamage3D::InitializeMaterial(
+void SmallStrainIsotropicDamage3D::FinalizeMaterialResponseCauchy(
+    ConstitutiveLaw::Parameters& rParametersValues)
+{
+    FinalizeMaterialResponsePK2(rParametersValues);
```
this does not look correct to me. 

i would call the correct one in here... (i mean, ending with PK2, Cauchy...)

rubenzorrilla committed:
Add common base class

rubenzorrilla committed:
Minor test correction (transpose)

rubenzorrilla committed:
Improve documentation

sunethwarna committed:
changing the clear method

sunethwarna committed:
changing the clear method

rubenzorrilla committed:
Missing overrides

roigcarlo committed:
Enable full test output

RiccardoRossi committed:
attempting to fix problem in win

RiccardoRossi committed:
minimal correction

RiccardoRossi committed:
recorrecting a wrong correction

sunethwarna committed:
Merge pull request #8550 from KratosMultiphysics/core/sensitivity_builder/change_clear

[Core] Fixing Clear in Sensitivity Builder

roigcarlo committed:
Undo changes

rubenzorrilla committed:
Merge branch 'master' into linearsolvers/eigen-svd-decomposition

AlejandroCornejo committed:
correcting DEM procedures

AlejandroCornejo committed:
solving 2 femdem

AlejandroCornejo committed:
update tests

AlejandroCornejo committed:
more

AlejandroCornejo committed:
updating the deprecated method

rubenzorrilla committed:
[TEST CI] Add together with linear solvers

rubenzorrilla committed:
[TEST CI] Add together with linear solvers

rubenzorrilla committed:
Merge branch 'linearsolvers/eigen-svd-decomposition' of https://github.com/KratosMultiphysics/Kratos into linearsolvers/eigen-svd-decomposition

AlejandroCornejo committed:
Merge pull request #8552 from KratosMultiphysics/solve-femdem-test-failing

[FEMDEM] Updating App to deprecated methods in DEM & minor solving in DEM

Commit:
Change order as suggested

philbucher committed:
Merge remote-tracking branch 'origin/master' into linearsolvers/eigen-svd-decomposition

philbucher committed:
resore changes

philbucher committed:
move exclusions

RiccardoRossi committed:
minor improvements

rubenzorrilla committed:
Update applications/LinearSolversApplication/tests/test_eigen_dense_decompositions.py

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

rubenzorrilla committed:
Rename settings

rubenzorrilla committed:
Update applications/LinearSolversApplication/tests/test_eigen_dense_decompositions.py

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

philbucher committed:
Merge pull request #8523 from KratosMultiphysics/mpi/deprecate-mpcommutils-without-datacomm

[MPI][ModelPartCommunicatorUtilities] deprecate SetMPICommunicator without DataCommunicator

philbucher committed:
Merge pull request #8539 from KratosMultiphysics/ci/add-intel-to-docker

[CI] add Intel compiler to Docker image

philbucher committed:
not installing recommendations

philbucher committed:
update gfortran version

rubenzorrilla committed:
Merge pull request #8546 from KratosMultiphysics/linearsolvers/eigen-svd-decomposition

[LinearSolvers] Add Eigen SVDs

rubenzorrilla committed:
Merge branch 'master' into fsi/two-fields-coupling

philbucher committed:
enable mkl again

philbucher committed:
try to use global fortran compiler

philbucher committed:
Merge pull request #8560 from KratosMultiphysics/ci/docker-not-install-recommendations

[CI][DockerImage] not installing recommendations

philbucher committed:
adding missing llvm packages

Commit:
Merge pull request #8553 from KratosMultiphysics/core/tessellation_modeler_geom

[Core] Change order in geometry data

Commit:
Merge branch 'master' into core/cad_tessellation_modeler

philbucher committed:
removing valgrind

philbucher committed:
directly push new image

philbucher committed:
Revert hotfix

philbucher committed:
Update DockerFile

philbucher committed:
Merge pull request #8564 from KratosMultiphysics/ci/docker-missing-llvm-packages

[CI-Docker] adding missing llvm packages

philbucher committed:
Merge branch 'master' into ci/mkl-enable-again

philbucher committed:
hardcodeing sourcing of Intel

philbucher committed:
wrong shell

philbucher committed:
change to bash

philbucher committed:
only for gcc

philbucher committed:
explicitly specifying which clang to use
sneaky intel changes the default clang!

philbucher committed:
source globally

Rbravo555 committed:
skip test if FluidDynamicsApp not compiled (or no numpy) (#8554)

* skip test if FluidDynamicsApp not compiled (or no numpy)

* using skipIfApplicationsNotAvailable in Rom tests

* USING skipUnless(numpy_available), INSTEAD OF skipIf(numpy_available==False)

Commit:
suggested changes

philbucher committed:
specify explicitly which MPI to use

philbucher committed:
using MPIEXEC_EXECUTABLE

philbucher committed:
simplification

philbucher committed:
fix name

philbucher committed:
specify mpi compiler
when using GCC mpi cannot be found for some reason

philbucher committed:
removing IntelMPI

until issue with selecting MPI is fixed

loumalouomega committed:
Update test_line_2d_2.cpp

loumalouomega committed:
Update test_triangle_3d_3.cpp

loumalouomega committed:
Update test_quadrilateral_3d_4.cpp

philbucher committed:
make entrypoint executable

miguelmaso committed:
Merge pull request #8547 from KratosMultiphysics/shallow/custom-output-for-running-examples

[Shallow] Custom output for academic examples

philbucher committed:
Merge pull request #8565 from KratosMultiphysics/ci/docker-remove-intel-mpi

removing IntelMPI from DockerImage

philbucher committed:
Merge remote-tracking branch 'origin/master' into ci/mkl-enable-again

philbucher committed:
not specifying MPI

roigcarlo committed:
Inheriting ownership from move constructor

This should fix the windows problems. Still investigating why the move behavior is different between platforms

philbucher committed:
specifying compiler explicitly

loumalouomega committed:
Merge pull request #8566 from KratosMultiphysics/core/fix-intel-warnings

philbucher committed:
proper loading of Intel env for MKL

philbucher committed:
fixing shell in nightly build

marcnunezc committed:
Disable mmg/parmmg warning and reduce progress outputs

Commit:
Merge pull request #8458 from KratosMultiphysics/core/cad_tessellation_modeler

[Core] Cad Tessellation Modeler 2/2

philbucher committed:
try to call entrypoint scripts manually

philbucher committed:
show files

philbucher committed:
root

philbucher committed:
bash

philbucher committed:
revert, seems like the entrypoint doesn't want to work :/

marcnunezc committed:
Merge pull request #8567 from KratosMultiphysics/ci/reduce-mmg-log

[CI] Disable mmg/parmmg warnings and reducing progress outputs

philbucher committed:
Merge pull request #8563 from KratosMultiphysics/ci/mkl-enable-again

[CI] enable MKL again after 8539

roigcarlo committed:
Previnting use of elated constructor in move returns

philbucher committed:
enable paralle build

sunethwarna committed:
update HDF5App Readme

sunethwarna committed:
minor

tteschemacher committed:
clean up comments

tteschemacher committed:
update constructors

tteschemacher committed:
Add ProjectionPoint

tteschemacher committed:
Remove comments and add Center

tteschemacher committed:
Remove comments and add ProjectionPoint

tteschemacher committed:
update projection_nurbs_geometry_utilities.h

tteschemacher committed:
Update nurbs geometries

tteschemacher committed:
update cad model reading

tteschemacher committed:
temporary hard coded check

tteschemacher committed:
return DomainInterval

philbucher committed:
changes to configure file should trigger nightly build

philbucher committed:
initial add

philbucher committed:
fix formatting

philbucher committed:
fix syntax

RiccardoRossi committed:
using pointers to prevent problems with std move

RiccardoRossi committed:
Merge branch 'use_span_in_csr_internals' of https://github.com/KratosMultiphysics/Kratos into use_span_in_csr_internals

RiccardoRossi committed:
small cleanup

RiccardoRossi committed:
corrections. should now pass tests

RiccardoRossi committed:
trying to make centos old gcc happy
(code should be already valid)

RiccardoRossi committed:
next attempt at making centos happy

RiccardoRossi committed:
using TIndexType everywhere to attempt
making old gcc happy

loumalouomega committed:
Update SimpleMortarMapperProcess

loumalouomega committed:
Update ConstraintUtilities

loumalouomega committed:
Update DeflationUtilities

loumalouomega committed:
Update MortarUtilities

loumalouomega committed:
Update B&S elimination

loumalouomega committed:
Update B&S elimination

loumalouomega committed:
Update B&S with LM

loumalouomega committed:
Update B&S bllock

loumalouomega committed:
Update SparseMatrixMultiplication

loumalouomega committed:
Update I

loumalouomega committed:
Update II

loumalouomega committed:
Update III

loumalouomega committed:
Update IV

loumalouomega committed:
Update V

loumalouomega committed:
Update VI

loumalouomega committed:
Update VII

loumalouomega committed:
Update VIII

loumalouomega committed:
Update IX

loumalouomega committed:
Ups

rubenzorrilla committed:
Merge branch 'master' into fsi/two-fields-coupling

tteschemacher committed:
Update brep_curve_on_surface.h

loumalouomega committed:
Merge pull request #8575 from KratosMultiphysics/contact/update-atomic-utilities

loumalouomega committed:
Merge pull request #8574 from KratosMultiphysics/core/missing-atomic-utilities

philbucher committed:
missing initialization

philbucher committed:
wrong way round

loumalouomega committed:
Using format (I)

loumalouomega committed:
Using format (II)

loumalouomega committed:
Using format (III)

loumalouomega committed:
It is external

loumalouomega committed:
Using format (IV)

loumalouomega committed:
Proper format

RiccardoRossi committed:
Update kratos/utilities/amgcl_csr_spmm_utilities.h

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

RiccardoRossi committed:
removign unneeded file

RiccardoRossi committed:
Merge branch 'use_span_in_csr_internals' of https://github.com/KratosMultiphysics/Kratos into use_span_in_csr_internals

RiccardoRossi committed:
Merge branch 'master' into use_span_in_csr_internals

loumalouomega committed:
Correct

loumalouomega committed:
Minor

RiccardoRossi committed:
Merge branch 'use_span_in_csr_internals' of https://github.com/KratosMultiphysics/Kratos into use_span_in_csr_internals

tteschemacher committed:
Fix constructor

tteschemacher committed:
use direct int to bool cast

tteschemacher committed:
direct bool to int cast

tteschemacher committed:
direct bool to int cast

tteschemacher committed:
Add comment

tteschemacher committed:
Add comment to ProjectionPoint

rubenzorrilla committed:
Minor fix in tests

rubenzorrilla committed:
Update Python tests

philbucher committed:
Merge pull request #8571 from KratosMultiphysics/ci/parallel-build

[CI] enable parallel build

philbucher committed:
remove usage of ccache

philbucher committed:
Merge remote-tracking branch 'origin/master' into ci/without-unity

philbucher committed:
removing old comments

philbucher committed:
Merge pull request #8522 from KratosMultiphysics/trilinos/create-epetra-comm-from-data-comm

[Trilinos] adding CreateEpetraCommunicator with DataCommunicator

rubenzorrilla committed:
Clean up interface mesh creation

philbucher committed:
adding intel build separately

rubenzorrilla committed:
Remove no longer used methods

philbucher committed:
updated StructuralMechanics

philbucher committed:
updated MeshMoving

philbucher committed:
updated FluidDynamics

philbucher committed:
cleaning and updating tests

philbucher committed:
disable runkratos

philbucher committed:
deprecating move constructor

philbucher committed:
fix paths

joaquinirazabal committed:
[DEM] Add tables to processes (#8559)

* Add tables to forces and moments process

* Clean-up

* Add tables to kinematic_constraints_process

* Create new process

* Update python process

* Auto stash before merge of "dem/apply_kinematic_constraints_walls_process" and "dem/add-tables-to-processes"

* Update processes after adding element to submodelpart

* Minor

* Change h files to hpp

* Delete flux_data_new.hdf5

* Add null to values and tables

philbucher committed:
disable deprecation warnings in Win build
How was this working before?

philbucher committed:
enable Metis & TrilinosApp

philbucher committed:
Merge pull request #8584 from KratosMultiphysics/apps/update-trilinos-comm-usage

[Apps] update usage of EpetraComm construction after #8522

philbucher committed:
adding only Metis

philbucher committed:
skiptest wasn't working

philbucher committed:
update a test

philbucher committed:
use default settingsadding default arg

rubenzorrilla committed:
Merge branch 'master' into fsi/two-fields-coupling

miguelmaso committed:
update problemtype

miguelmaso committed:
shock capturing parameters

miguelmaso committed:
update topography process

miguelmaso committed:
update topography process 2

miguelmaso committed:
add L2 norm utility

miguelmaso committed:
update convergence output

miguelmaso committed:
fix test

philbucher committed:
Merge pull request #8581 from KratosMultiphysics/core/remove-old-comments

[Core] removing old comments

philbucher committed:
fix order

philbucher committed:
using ccache for non-unity build

philbucher committed:
try with clang

philbucher committed:
now it works

rubenzorrilla committed:
Body-fitted clean up

rubenzorrilla committed:
Embedded solver clean up

miguelmaso committed:
Merge pull request #8590 from KratosMultiphysics/shallow/aux-developments

Shallow/aux developments

RiccardoRossi committed:
making explicit that we are using Kratos::span

RiccardoRossi committed:
minor cleanups

RiccardoRossi committed:
avoiding passing default_comm

tteschemacher committed:
Merge pull request #8572 from KratosMultiphysics/core/brep_curve_o_s_advancements

[Core] General enhancements for Brep and Nurbs geometries

philbucher committed:
Merge pull request #8586 from KratosMultiphysics/core/lock-obj-depr-move-ctor

[Core][LockObject] deprecating move constructor

miguelmaso committed:
update benchmark

rubenzorrilla committed:
Update applications/FSIApplication/custom_utilities/partitioned_fsi_utilities.hpp

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

rubenzorrilla committed:
Update applications/FSIApplication/custom_utilities/partitioned_fsi_utilities.hpp

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

rubenzorrilla committed:
Update applications/FSIApplication/custom_utilities/partitioned_fsi_utilities.hpp

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

rubenzorrilla committed:
Update applications/FSIApplication/python_scripts/fsi_coupling_interface.py

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

rubenzorrilla committed:
Update applications/FSIApplication/python_scripts/fsi_coupling_interface.py

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

rubenzorrilla committed:
Update applications/FSIApplication/tests/cpp_tests/test_partitioned_fsi_utilities.cpp

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

rubenzorrilla committed:
Merge branch 'master' into fsi/two-fields-coupling

RiccardoRossi committed:
Merge pull request #8279 from KratosMultiphysics/use_span_in_csr_internals

enabling spmm and many corrections

philbucher committed:
Merge pull request #8573 from KratosMultiphysics/ci/without-unity

[CI] add Core build without Unity

philbucher committed:
Merge pull request #8587 from KratosMultiphysics/mpi/fix-skiptest-metis

[MPI] fix skipping of tests if Metis is not available

philbucher committed:
Merge branch 'master' into ci/separate-icc-build

philbucher committed:
disable metis

rubenzorrilla committed:
Code review and Codacy

philbucher committed:
Merge pull request #8578 from KratosMultiphysics/core/hotfix-missing-init

[Core] missing initialization

philbucher committed:
Merge pull request #8582 from KratosMultiphysics/ci/separate-icc-build

[CI] ICC build V3

rubenzorrilla committed:
Merge pull request #8561 from KratosMultiphysics/fsi/two-fields-coupling

[FSI] Two fields coupling and required refactoring

sunethwarna committed:
Update applications/HDF5Application/python_scripts/core/operations/model_part.py

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

sunethwarna committed:
Update applications/HDF5Application/python_scripts/core/operations/model_part.py

Co-authored-by: Philipp Bucher <philipp.bucher@tum.de>

sunethwarna committed:
change to gauss_points

sunethwarna committed:
Merge branch 'hdf5/feature/add_integration_points_output' of https://github.com/KratosMultiphysics/Kratos into hdf5/feature/add_integration_points_output

sunethwarna committed:
change read me

miguelmaso committed:
Merge pull request #8597 from KratosMultiphysics/shallow/benchmark-update

[Shallow] update benchmark

sunethwarna committed:
Merge pull request #8537 from KratosMultiphysics/hdf5/feature/add_integration_points_output

[HDF5] Add integration points output

philbucher committed:
Merge pull request #8585 from KratosMultiphysics/trilinos/update-tests

[Trilinos] cleaning and updating tests

marandra committed:
Add noalias

marandra committed:
Remove unused measures

marandra committed:
Add more noalias

marandra committed:
Fix merge conflicts

marandra committed:
Fix merge conflicts
